integration-sandbox/packages/contracts/test/index.test.ts

```diff
@@ -71,21 +71,21 @@ describe("arrays", async () => {
     await expect(pushManyButton).toBeVisible();
     await expect(pushOneButton).toBeVisible();
 
-    // make sure scripts are loaded
-    await page.waitForLoadState("domcontentloaded");
```

Comment:
Did you try `load` instead of `domcontentloaded`?

afaik `domcontentloaded` is when the DOM itself is ready (you can manipulate the DOM in scripts), but `load` waits for all resources to load/execute (scripts, styles, images, etc.)
https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event

Reply:
tried some events in https://github.com/latticexyz/mud/pull/915 a couple weeks ago, don't remember which at this point. I can check tomorrow

Reply:
`load` seems to have worked! feels like this will scale better as we add more tests, too